dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -219,9 +185,31 @@ RUN cd {{$beatHome}}/.node \
      esac \
   && mkdir -p node \
   && curl ${NODE_DOWNLOAD_URL} | tar -xJ --strip 1 -C node \
-  && chmod ug+rwX -R $NODE_PATH \
-  && npm i -g -f @elastic/synthetics && chmod ug+rwX -R $NODE_PATH
+  && chmod ug+rwX -R $NODE_PATH
+
+# Install synthetics as a regular user, installing npm deps as root odesn't work
+RUN chown -R {{ .user }} $NODE_PATH
+USER {{ .user }}
+# If this fails dump the NPM logs
+RUN npm i -g --loglevel verbose -f @elastic/synthetics || sh -c 'tail -n +1 /root/.npm/_logs/* && exit 1'
+RUN chmod ug+rwX -R $NODE_PATH 
+USER root
+
+# Install the deps as needed by the exact version of playwright elastic synthetics uses
+# We don't use npx playwright install-deps because that could pull a newer version
+# Install additional fonts as well
+RUN for iter in {1..10}; do \
+    apt-get update -y && \
+    $NODE_PATH/node/lib/node_modules/@elastic/synthetics/node_modules/.bin/playwright install-deps && \
```

Comment:
Adding it here, so we don't miss this change - https://github.com/elastic/beats/pull/32122/files#r928044452